Storage/api.h

```diff
@@ -219,6 +219,13 @@ STORAGE_HANDLERS_MAP_ENTRY GenerateRESTfulHandler(STORAGE& storage, const std::s
 }
 
 }  // namespace impl
+template <bool CALL>
```

Comment:
@nyospe Yes un-reversing the boolean logic would make sense. How about `CallIf` or `ExecuteIf` as the name instead? And +1 to Max suggesting this belongs in `Bricks/template`.